src/http/websocket_http_client.zig

```diff
@@ -954,7 +954,9 @@ pub fn NewWebSocketClient(comptime ssl: bool) type {
         const WebSocket = @This();
 
         pub usingnamespace bun.New(@This());
-
+        fn isDeadPtr(ptr: *WebSocket) bool {
+            return @intFromPtr(ptr) == 1;
```

Comment:
same thing here. we should not have this method

Reply:
✂️ thanks to catch it removed